nim_status/lib/accounts.nim

```diff
@@ -0,0 +1,44 @@
+import os, json
+import sqlcipher, web3, chronos, json_serialization
+import settings, database, callrpc
+
+var db_conn*: DbConn
+var web3_conn*: Web3
+
+proc login*(accountData, password: string) =
+  # TODO get account, validate password, etc
+  # TODO: should this be async?
+  # TODO: db should have been initialized somewhere, not here
+  # TODO: determine where will the DB connection live. In the meantime I'm storing it into a global variable
+  # TODO: determine where the web3 conn will live
+
+  let path = currentSourcePath.parentDir().parentDir().parentDir() & "/build/" & accountData
```

Comment:
I forgot about this earlier, but then thought to check it again when I saw builds failing in GHA. I don't think it's causing the errors currently reported in GHA, but I still think we should probably change the path to be the current working directory until we've implemented a solution for #117.